flathub.json

```diff
@@ -1,4 +1 @@
-{
-    "only-arches": ["x86_64"],
-    "automerge-flathubbot-prs": true
-}
+{"only-arches": ["x86_64"]}
```

Comment:
this remain disappointing. I thought the goal for using bloat like Electron was to support many platform and you just do the opposite.

Reply:
1. I have suggested the use of Tauri and [there is a PR for that](https://kolaente.dev/vikunja/desktop/pulls/127).
2. I am not the maintainer of the project
3. Please be more polite. All of your comments have an aggressive tone which can be both offending and can be discouraging.

Reply:
@RaptaG well I got that answer initially https://github.com/flathub/flathub/pull/4319#discussion_r1258589072

Electron DO support `aarch64`.